dt-people-groups/people-groups-base.php

```diff
@@ -32,6 +31,58 @@ public function __construct( $is_admin = true ) {
         add_action( 'dt_details_additional_section', [ $this, 'dt_details_additional_section' ], 40, 2 );
     }
 
+    public function register_post_type() {
```

Comment:
@kodinkat , 
The post type is registered on line 118. We don't want to register it twice?